config/default.yml

```diff
@@ -164,7 +170,7 @@ RSpec/DescribeClass:
       - mailbox
       - aruba
   VersionAdded: '1.0'
-  VersionChanged: '1.44'
+  VersionChanged: '3.0'
```

Comment:
I was following the guidance in the PR checklist to "Set VersionChanged in config/default.yml to the next major version" for a "changed" cop. That said, I also agree that it's more of a patch-level change. I agree it could be considered a bug fix to resolve the issue that the expected `type`s won't be excluded unless the spec includes the `type` metadata. Should I go ahead and update the `VersionChanged` to 2.4.1?

Reply:
"What might break is if the project tries to stick with controller classes in request specs" - I will test this...